apps/dashboard/app/(app)/logs-v2/components/controls/components/logs-filters/components/methods-filter.tsx

```
@@ -60,17 +56,36 @@ export const MethodsFilter = () => {
-        </label>
+        <div className="flex justify-between items-center">
+          <label className="flex items-center gap-2 cursor-pointer">
+            <Checkbox
+              checked={checkboxes.every((checkbox) => checkbox.checked)}
+              className="size-[14px] rounded border-gray-4 [&_svg]:size-3"
+              onClick={handleSelectAll}
+            />
+            <span className="text-xs text-accent-12 ml-2">
+              {checkboxes.every((checkbox) => checkbox.checked) ? "Unselect All" : "Select All"}
+            </span>
+          </label>
```

Biome (1.9.4)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)